Client/Frontend/Brave Today/Cards/PartnerCardView.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import BraveUI
+
+class PartnerCardView: FeedCardBackgroundButton, FeedCardContent {
+    var actionHandler: ((Int, FeedItemAction) -> Void)?
```

Comment:
this first parameter is unclear for me, you pass `0` to it in few places, maybe make this param named here for better readability?

Reply:
These are explained in the `FeedCardContent` protocol declaration